lib/metadata/wrapper.js

```diff
@@ -67,54 +67,25 @@ const metadata = {
         });
     },
 
-    putObjectMD: (bucketName, objName, objVal, log, cb, params) => {
+    putObjectMD: (bucketName, objName, objVal, params, log, cb) => {
         log.debug('putting object in metdata');
-        client.putObject(bucketName, objName, objVal, log, err => {
-            if (err) {
```

Comment:
I would prefer keeping the error logs in the wrapper so we are certain we are logging them.